packages/cozy-harvest-lib/src/components/MountPointContext.jsx

```diff
@@ -23,7 +23,7 @@ export class RawMountPointProvider extends React.Component {
       .split('/')
       .concat(route.split('/'))
       .filter(Boolean)
-    history.push(`/${segments.join('/')}`)
+    history.replace(`/${segments.join('/')}`)
```

Comment:
We can not have a method called `pushHistory` now that we `replaceHistory`. Can you rename it please?

Reply:
> We can not have a method called `pushHistory` now that we `replaceHistory`. Can you rename it please?

yes of course, I should even create pushHistory back, don't you think?
but it will not be used by anything for now..

Reply:
let's not create unused method. YAGNI.

Reply:
Updated